Buildings/Controls/OBC/CDL/Logical/Proof.mo

```diff
@@ -0,0 +1,251 @@
+within Buildings.Controls.OBC.CDL.Logical;
+block Proof "Verify two boolean inputs"
+  parameter Real validDelay(
+    final quantity="Time",
+    final unit="s")
+    "Delay to valid input. The input must remain unchanged in the time";
```

Comment:
must remain unchanged for what to happen? This is not clear.

Reply:
Thanks for revising it.